content/ghc-9.12-cabal-3.14/index.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "GHC 9.12 & cabal 3.14 releases"
+date = 2024-12-22
+[taxonomies]
+authors = ["Hécate"]
+categories = ["Cabal", "GHC"]
+tags = ["Release"]
++++
+
+To conclude the year 2024, the GHC and Cabal teams are happy to announce the release of GHC 9.12 and cabal 3.14.
+
+<!-- more -->
+
+
+Here are some highlights:
+
+## GHC 9.12
+
+### Language Extensions
+
+This release brings its lot of new and exciting extensions:
```

Comment:
```suggestion
This release brings many new and exciting extensions:
```
"its lot of" was wrong anyway; "a lot of" would be more correct, but feels too casual for a release announcement.

Reply:
Agreed.

Reply:
I thought "lot" as a noun could be used, like in the sentence "The lot includes samples of different products."

Reply:
It can be, but this is not at all a common usage; "lot" there usually refers to a shipment. I'm aware of a few Commonwealth idioms using it that way, but those don't seem to really fit here either.

Reply:
Got it, thanks!

Reply:
Someone who speaks British/Commonwealth English might be able to add more. I'd avoid it just because it looks fairly odd to American English speakers.

Reply:
In contemporary American English "a lot" is informal for "many". I'm not a native but I write texts for a job. I'd suggest using "many" in release notes and alike.